<ui version="4.0" >
 <class>InstallTalkFrm</class>
 <widget class="QDialog" name="InstallTalkFrm" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Install Talk Files</string>
  </property>
  <layout class="QGridLayout" >
   <item rowspan="6" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="rbutilqt.qrc" >:/icons/icons/wizard.xpm</pixmap>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Select the Folder to generate Talkfiles for.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2" >
    <widget class="QLineEdit" name="lineTalkFolder" />
   </item>
   <item row="1" column="3" >
    <widget class="QPushButton" name="buttonBrowse" >
     <property name="text" >
      <string>&amp;Browse</string>
     </property>
     <property name="icon" >
      <iconset resource="rbutilqt.qrc" >:/icons/icons/system-search.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Generation settings</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="labelEncProfile" >
        <property name="text" >
         <string>Encoder profile:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelTtsProfile" >
        <property name="text" >
         <string>TTS profile:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" colspan="3" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Generation options</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="OverwriteWav" >
        <property name="text" >
         <string>Overwrite Wavefiles</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="RemoveWav" >
        <property name="text" >
         <string>Remove Wavefiles</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="recursive" >
        <property name="text" >
         <string>Run recursive</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QCheckBox" name="StripExtensions" >
        <property name="text" >
         <string>Strip Extensions</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" >
       <widget class="QCheckBox" name="OverwriteTalk" >
        <property name="text" >
         <string>Overwrite Talkfiles</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;Install</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >:/icons/icons/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="icon" >
        <iconset resource="rbutilqt.qrc" >:/icons/icons/process-stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rbutilqt.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>InstallTalkFrm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>472</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel" >
     <x>382</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>InstallTalkFrm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>561</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel" >
     <x>522</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
